### PR TITLE
Add cernbox.app v2.5.4.2623

### PR DESCRIPTION
**Pull request by adavidzh**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) ~~or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).~~

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

Please have a look at the uninstall section; I am not completely sure it is the best it can be.

### CODE REVIEW

File: Casks/cernbox.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask "cernbox" do
+  version "2.5.4.2623"
+  sha256 "a54a5d7afdd918945da482041f85f4afb1db0bc512829dbe85001abb78fafff1"
+
+  url "https://cernbox.cern.ch/cernbox/doc/MacOSX/cernbox-#{version}.pkg"
+  name "CERNBox Client"
+  desc "Acccess to cloud storage for users with a regular CERN computing account"
```
Comment (miccal) [inline Casks/cernbox.rb:7]:
```suggestion
  desc "Cloud storage for CERN users"
```

File: Casks/cernbox.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  homepage "https://cernbox.web.cern.ch/cernbox/"
+
+  pkg "cernbox-#{version}.pkg"
+
+  # uninstall quit: [ "ch.cern.cernbox" ]
+  # The above did not work with:
+  # Warning: Application 'ch.cern.cernbox' did not quit. Enable Automation Access for "Terminal > System Events" in
+  # "System Preferences > Security > Privacy > Automation" if you haven't already.
+
+  uninstall signal:     [["TERM", "ch.cern.cernbox"]],
```
Comment (miccal) [inline Casks/cernbox.rb:17]: The `.pkg` runs a preinstall script `cernbox-2.5.4.2623.pkg/CERNBox.pkg/Scripts/preinstall`, the contents of which is simply:
```
#!/bin/sh

# kill the old version. see issue #2044
killall cernbox

exit 0
```
which suggests why a regular `uninstall quit:` failed.

File: Casks/cernbox.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+cask "cernbox" do
+  version "2.5.4.2623"
+  sha256 "a54a5d7afdd918945da482041f85f4afb1db0bc512829dbe85001abb78fafff1"
+
+  url "https://cernbox.cern.ch/cernbox/doc/MacOSX/cernbox-#{version}.pkg"
```
Comment (miccal) [inline Casks/cernbox.rb:5]: Please add the following `appcast`:
```
appcast "https://cernbox.web.cern.ch/cernbox/downloads/"
```

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/cernbox.rb (hunk 3 above)
Comment (adavidzh) [reply Casks/cernbox.rb:5]: Added in 5882309f1a1d05ace5e7937c9770a599c67bf025

**Comment by adavidzh** [comment]
While adding the `appcast` stanza I have found that there are also other versions, including a `latest` link at [this URL](https://cernbox.cern.ch/cernbox/doc/MacOSX/). Could that URL be put to good use?

**Comment by miccal** [comment]
> While adding the `appcast` stanza I have found that there are also other versions, including a `latest` link at [this URL](https://cernbox.cern.ch/cernbox/doc/MacOSX/). Could that URL be put to good use?

Yes it could -- please change the  `appcast` to:
```
appcast "https://cernbox.cern.ch/cernbox/doc/MacOSX/latest.link"
```

**Comment by miccal** [comment]
> While adding the `appcast` stanza I have found that there are also other versions, including a `latest` link at [this URL](https://cernbox.cern.ch/cernbox/doc/MacOSX/). Could that URL be put to good use?

That link seems to list pre-releases, and the `latest` link has not been updated since 2015, so I do not think so.

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @adavidzh.

**Comment by miccal** [comment]
@reitermarkus and @vitorgalvao is there a way I can approve something without the Bot auto-merging?

**Comment by vitorgalvao** [comment]
> is there a way I can approve something without the Bot auto-merging?

Nope (I think). I agree there are certain situations when it’s inconvenient, but to disable it we’d lose the situations where it is convenient.

**Comment by reitermarkus** [comment]
You can add the `do not merge` label.

**Comment by reitermarkus** [comment]
We an also make the `awaiting` labels behave that way.

**Comment by vitorgalvao** [comment]
> You can add the `do not merge` label.

Hadn’t thought of that. Nice thinking.

> We can also make the `awaiting` labels behave that way.

Just `awaiting-maintainer-feedback` should do it, I’d say.

**Pull request merged**